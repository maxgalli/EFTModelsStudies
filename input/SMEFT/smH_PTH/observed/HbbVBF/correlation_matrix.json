{
    "r_smH_PTH_450_500": {
        "r_smH_PTH_450_500": 1.0,
        "r_smH_PTH_500_550": 0.13455701182408475,
        "r_smH_PTH_550_600": 0.08029517201284368,
        "r_smH_PTH_600_675": 0.08161942571206743,
        "r_smH_PTH_675_800": -0.009393344842103031,
        "r_smH_PTH_800_1200": -0.0544787072694425
    },
    "r_smH_PTH_500_550": {
        "r_smH_PTH_450_500": 0.13455701182408475,
        "r_smH_PTH_500_550": 1.0,
        "r_smH_PTH_550_600": 0.09007311587695675,
        "r_smH_PTH_600_675": 0.06796358275839676,
        "r_smH_PTH_675_800": -0.010294917316844983,
        "r_smH_PTH_800_1200": -0.030468228255362925
    },
    "r_smH_PTH_550_600": {
        "r_smH_PTH_450_500": 0.08029517201284368,
        "r_smH_PTH_500_550": 0.09007311587695675,
        "r_smH_PTH_550_600": 1.0,
        "r_smH_PTH_600_675": 0.18747753567596745,
        "r_smH_PTH_675_800": -0.09016548612952294,
        "r_smH_PTH_800_1200": -0.07819594064927393
    },
    "r_smH_PTH_600_675": {
        "r_smH_PTH_450_500": 0.08161942571206743,
        "r_smH_PTH_500_550": 0.06796358275839676,
        "r_smH_PTH_550_600": 0.18747753567596745,
        "r_smH_PTH_600_675": 1.0,
        "r_smH_PTH_675_800": -0.06118710004645922,
        "r_smH_PTH_800_1200": -0.05563536658003437
    },
    "r_smH_PTH_675_800": {
        "r_smH_PTH_450_500": -0.009393344842103031,
        "r_smH_PTH_500_550": -0.010294917316844983,
        "r_smH_PTH_550_600": -0.09016548612952294,
        "r_smH_PTH_600_675": -0.06118710004645922,
        "r_smH_PTH_675_800": 1.0,
        "r_smH_PTH_800_1200": 0.0442999621246516
    },
    "r_smH_PTH_800_1200": {
        "r_smH_PTH_450_500": -0.0544787072694425,
        "r_smH_PTH_500_550": -0.030468228255362925,
        "r_smH_PTH_550_600": -0.07819594064927393,
        "r_smH_PTH_600_675": -0.05563536658003437,
        "r_smH_PTH_675_800": 0.0442999621246516,
        "r_smH_PTH_800_1200": 1.0
    }
}