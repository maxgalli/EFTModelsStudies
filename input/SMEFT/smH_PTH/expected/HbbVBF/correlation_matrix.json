{
    "r_smH_PTH_450_500": {
        "r_smH_PTH_450_500": 1.0,
        "r_smH_PTH_500_550": 0.13522338031566572,
        "r_smH_PTH_550_600": 0.09524920088146817,
        "r_smH_PTH_600_675": 0.07357196348522804,
        "r_smH_PTH_675_800": 0.04955572086659819,
        "r_smH_PTH_800_1200": 0.032542043858606234
    },
    "r_smH_PTH_500_550": {
        "r_smH_PTH_450_500": 0.13522338031566572,
        "r_smH_PTH_500_550": 1.0,
        "r_smH_PTH_550_600": 0.0766826529486654,
        "r_smH_PTH_600_675": 0.06044753054453216,
        "r_smH_PTH_675_800": 0.04541589381721617,
        "r_smH_PTH_800_1200": 0.029892430594985846
    },
    "r_smH_PTH_550_600": {
        "r_smH_PTH_450_500": 0.09524920088146817,
        "r_smH_PTH_500_550": 0.0766826529486654,
        "r_smH_PTH_550_600": 1.0,
        "r_smH_PTH_600_675": 0.049535513518313604,
        "r_smH_PTH_675_800": 0.039032903136133286,
        "r_smH_PTH_800_1200": 0.026288497889517308
    },
    "r_smH_PTH_600_675": {
        "r_smH_PTH_450_500": 0.07357196348522804,
        "r_smH_PTH_500_550": 0.06044753054453216,
        "r_smH_PTH_550_600": 0.049535513518313604,
        "r_smH_PTH_600_675": 1.0,
        "r_smH_PTH_675_800": 0.041561351874604475,
        "r_smH_PTH_800_1200": 0.024550867615309304
    },
    "r_smH_PTH_675_800": {
        "r_smH_PTH_450_500": 0.04955572086659819,
        "r_smH_PTH_500_550": 0.04541589381721617,
        "r_smH_PTH_550_600": 0.039032903136133286,
        "r_smH_PTH_600_675": 0.041561351874604475,
        "r_smH_PTH_675_800": 1.0,
        "r_smH_PTH_800_1200": 0.022453703102967352
    },
    "r_smH_PTH_800_1200": {
        "r_smH_PTH_450_500": 0.032542043858606234,
        "r_smH_PTH_500_550": 0.029892430594985846,
        "r_smH_PTH_550_600": 0.026288497889517308,
        "r_smH_PTH_600_675": 0.024550867615309304,
        "r_smH_PTH_675_800": 0.022453703102967352,
        "r_smH_PTH_800_1200": 1.0
    }
}